webpack.config.js

```diff
@@ -170,4 +170,13 @@ var config = {
   devtool: 'source-map'
 };
 
+if (process.env.NODE_ENV === 'production') {
```

Comment:
I'd probably have a comment that this will produce .gz files, which the server is configured to serve if detected on the filesystem